test/src/test/java/hudson/util/SecretCompatTest.java

```diff
@@ -82,6 +86,7 @@ public void encryptedValueStaysTheSameAfterRoundtrip() throws Exception {
     @Issue("SECURITY-304")
     @LocalData
     public void canReadPreSec304Secrets() throws Exception {
+        assumeTrue(runTestSometimes(NEVER_FAILING_TEST));
```

Comment:
Please no.

Reply:
I mistakenly thought that the "Please no" applied to all the changes in the pull request, but now that I read more carefully, I see that it applies to 4 tests that are specifically validating security fixes.  I'll remove the changes to tests related to security.  An undetected security regression is much, much more expensive than running the tests each time.

Thanks again for the review.

Reply:
Right, I specifically request security-related tests be exempted. Regular functional regressions being caught with 90+% likelihood seems mostly fine if the savings goal here is significant enough.